logger_test.go

```diff
@@ -184,19 +184,52 @@ func TestJSONLoggerLog(t *testing.T) {
 		logger.Log(DebugLevel, "foo")
 		assert.Equal(t, `{"level":"debug","msg":"foo"}`, buf.Stripped(), "Unexpected output from Log.")
 	})
+}
 
-	withJSONLogger(t, nil, func(logger Logger, buf *testBuffer) {
-		assert.Panics(t, func() { logger.Log(PanicLevel, "foo") }, "Expected logging at Panic level to panic.")
-		assert.Equal(t, `{"level":"panic","msg":"foo"}`, buf.Stripped(), "Unexpected output from panic-level Log.")
-	})
+func TestJSONLoggerLog_panic(t *testing.T) {
```

Comment:
I don't think that we're using this underscore convention in the rest of the test suite; purely for consistency, can we call these `TestJSONLoggerLogPanic` and `...Fatal`?